doc/leak_tracking/TROUBLESHOOT.md

```diff
@@ -128,6 +128,15 @@ To make debugging easier, invoke [ChangeNotifier.maybeDispatchObjectCreation]
 in constructor of the class. It will help
 to identify the owner in case of leaks.
 
+Check for compile time constant `kFlutterMemoryAllocationsEnabled` will make this
```

Comment:
"Be sure to guard the invocation behind the `kFlutterMemoryAllocationsEnabled` flag. This will ensure that `maybeDispatchObjectCreation` is only called if memory allocation events are enabled."

Also, can you add a quick note regarding how `kFlutterMemoryAllocationsEnabled` is set? Maybe link to the api docs for this const.